ocf_datapipes/transform/xarray/pv/create_pv_image.py

```diff
@@ -179,3 +206,40 @@ def _get_idx_of_pixel_closest_to_poi_geostationary(
         np.searchsorted(xr_data[y_dim_name].values[::-1], center_geostationary.y) - 1
     )
     return Location(x=x_index_at_center, y=y_index_at_center)
+
+
+def _normalize_by_pvlib(pv_system):
+    """
+    Normalize the output by pv_libs poa_global
+
+    Args:
+        pv_system: PV System in Xarray DataArray
+
+    Returns:
+        PV System in xarray DataArray, but normalized values
+    """
+    # TODO Add elevation
+    pvlib_loc = pvlib.location.Location(
+        latitude=pv_system.latitude.values, longitude=pv_system.longitude.values
+    )
+    times = pd.DatetimeIndex(pv_system.time_utc.values)
+    solar_position = pvlib_loc.get_solarposition(times=times)
+    clear_sky = pvlib_loc.get_clearsky(times)
+    print(pv_system.tilt.values)
+    print(pv_system.orientation.values)
+    total_irradiance = pvlib.irradiance.get_total_irradiance(
+        pv_system.tilt.values,
+        pv_system.orientation.values,
+        solar_zenith=solar_position["zenith"],
+        solar_azimuth=solar_position["azimuth"],
+        dni=clear_sky["dni"],
+        dhi=clear_sky["dhi"],
+        ghi=clear_sky["ghi"],
+    )
+    # Guess want fraction of total irradiance on panel, to get fraction to do with capacity
+    fraction_clear_sky = total_irradiance["poa_global"] / (
+        clear_sky["dni"] + clear_sky["dhi"] + clear_sky["ghi"]
+    )
+    pv_system /= pv_system.capacity_watt_power
+    pv_system *= fraction_clear_sky
+    return pv_system
```

Comment:
@simlmx here is the actual, working normalization, if that is what you meant by it. The fraction_clear_sky is between 0 and 1